### ISSUE TITLE
make adding new TDF data easier

### ISSUE THREAD

**Issue by AndrewLJackson** [healyke/SIDER#22]
add argument to prepareSider() to allow new TDF data to be provided by the user which is then tagged on to the end of the existing data before the various checks are performed.

**Comment by TGuillerme** [comment]
You mean to add to the isotopic data rather than replacing it? For now when you do:
```r
prepareSider(..., data.isotope = my_isotope_data, ...)
```
It uses user's isotope data table.
```r
prepareSider(..., data.isotope, ...) # left empty
```
It uses the inbuilt `isotope_data`. One easy solution for both could be:
```r
prepareSider(..., data.isotope = rbind(isotope_data, my_isotope_data), ...)
```

Did I got you right there?

**Comment by AndrewLJackson** [comment]
yes, something like this. I use the rbind() already in the vignette to do just this. we may decide this is not worth it, but i was thinking of a  `new.data` argument which would then get the `rbind` treatment internally
